Lib/compositor/font.py

```diff
@@ -6,6 +6,17 @@
 from compositor.error import CompositorError
 
 
+def _nameToString(nameRecord):
+    encoding = nameRecord.getEncoding()
+    try:
+        # python 3
+        ret = nameRecord.string.decode(encoding)
```

Comment:
We should add toStr() to the NameRecord in fonttools.

On 16-01-19 11:55 AM, Cosimo Lupo wrote:
> In Lib/compositor/font.py
> <https://github.com/typesupply/compositor/pull/10#discussion_r50098468>:
> 
>> @@ -6,6 +6,17 @@
>>  from compositor.error import CompositorError
>>  
>>  
>> +def _nameToString(nameRecord):
>> +    encoding = nameRecord.getEncoding()
>> +    try:
>> +        # python 3
>> +        ret = nameRecord.string.decode(encoding)
> 
> actually no. If in python2 you use |tostr| on a unicode string, it will
> reencode it to bytes using a default ascii encoding, which is not good here.
> 
> Maybe this would be better:
> 
> if str == bytes:
>     # python 2
>     ret = nameRecord.toBytes()
> else:
>     # python 3
>     ret = nameRecord.toUnicode()
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/typesupply/compositor/pull/10/files#r50098468>.
>